HIPs/hedera-hashgraph.md

```diff
@@ -0,0 +1,302 @@
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the JSON RPC Relay that today can be found at https://github.com/hashgraph/hedera-json-rpc-relay
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the Local Node that today can be found at https://github.com/hashgraph/hedera-local-node
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the Mirror Node that today can be found at https://github.com/hashgraph/hedera-mirror-node
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the Services that today can be found at https://github.com/hashgraph/hedera-services
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the JavaScript SDK that today can be found at https://github.com/hashgraph/hedera-sdk-js
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the Swift SDK that today can be found at https://github.com/hashgraph/hedera-sdk-swift
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the Rust SDK that today can be found at https://github.com/hashgraph/hedera-sdk-rust
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the Java SDK that today can be found at https://github.com/hashgraph/hedera-sdk-java
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the Go SDK that today can be found at
+  https://github.com/hashgraph/hedera-sdk-go
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the C++ SDK that today can be found at https://github.com/hashgraph/hedera-sdk-cpp
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the Solo CLI that today can be found at
+  https://github.com/hashgraph/solo
+
+As shown, the Hedera ecosystem already contains several sub-projects that must be part of `PROJECT`.
+All those sub-projects are necessary to deploy and run a full Hashgraph-based network and interact with that network.
+All mentioned projects are currently part of the Hashgraph organization at GitHub.
+Supplemental repositories include supporting tooling to deploy and validate a network built using the above codebases.
+In addition to the aforementioned repositories and projects, the community and ecosystem around Hedera and its unique
+hashgraph consensus algorithm is quite extensive, and we will want to transition a number of third-party projects
+coming from the community as an initial step once `PROJECT` has been set up. 
+An example of such a community project is https://github.com/OpenElements/hedera-enterprise.
+
+## GitHub issues
+The Hedera projects use GitHub issues today.
+Together, the projects contain more than 1,000 open issues that contain information that should be preserved since they
+contain great contributions from the community. Based on that, we propose to migrate the issues to the newly
+created repositories.
```

Comment:
"newly created repositories" - will we not directly transfer the repositories from the "hashgraph" organization to the new "PROJECT" organization? What happens with the existing repos?

Reply:
you are right. It makes more sense to transfer the repositories directly to LF